### PR TITLE
RLM-225 Updated rpc-upgrades jobs

### PR DESCRIPTION
**Pull request by cloudnull**
This introduces a few specific jobs which will ensure we have a good
baseline for AIO testing.

Signed-off-by: Kevin Carter <kevin.carter@rackspace.com>
Issue: https://rpc-openstack.atlassian.net/browse/RLM-225

Issue: [RLM-225](https://rpc-openstack.atlassian.net/browse/RLM-225)

### CODE REVIEW

File: rpc_jobs/irr_role_test.yml [hunk 1]
```diff
@@ -85,8 +85,14 @@
     #  release.
     context:
       - kilo:
-          # Kilo EOL SHA
-          series: "97e3425871659881201106d3e7fd406dc5bd8ff3"
+          irr_flavor: "performance2-15"
```
Comment (odyssey4me) [inline rpc_jobs/irr_role_test.yml]: This flavor is consistent, so why not just define it once in the image/series axis?

File: rpc_jobs/irr_role_test.yml [hunk 2]
```diff
@@ -80,14 +80,17 @@
     image:
       - trusty:
           IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+          irr_flavor: "performance2-15"
     # Context is used to define the origin "release" when leap upgrading
     #  The "series" parameter is used to define the checkout within the
     #  release.
     context:
+      - kilo
       - kilo:
```
Comment (odyssey4me) [inline rpc_jobs/irr_role_test.yml]: If you use the context of ``r11.0.0`` or ``kilo_r11.0.0`` then the series does not need to change in the job definition.

File: rpc_jobs/irr_role_test.yml [hunk 3]
```diff
@@ -80,14 +80,17 @@
...
-          # Kilo EOL SHA
-          series: "97e3425871659881201106d3e7fd406dc5bd8ff3"
-          irr_flavor: "performance2-15"
+          series: r11.0.0
+      - kilo:
+          series: r11.1.18
```
Comment (odyssey4me) [inline rpc_jobs/irr_role_test.yml]: This will result in the same ``kilo`` job being overwritten multiple times, resulting in only one left.

**Comment by cloudnull** [comment]
I added the issue to the PR and made sure its within the comments of the card.

**Comment by mattt416** [comment]
recheck_issue_link

**Pull request merged**